corehq/util/view_utils.py

```diff
@@ -69,7 +70,7 @@ def inner(request, *args, **kwargs):
         except BadRequest as e:
             return _get_json_exception_response(400, request, e)
         except Exception as e:
-            notify_exception(request, 'JSON exception response: {}'.format(e))
+            notify_exception(request, 'JSON exception response: {}'.format(smart_bytes(e)))
```

Comment:
I've used Django's `smart_bytes` to massage Unicode messages into something that dimagi-utils `notify_exception` won't choke on. Let me know if you think we should move `smart_bytes` [downstream to `notify_exception`](https://github.com/dimagi/dimagi-utils/blob/59f1116e120f3d9f81b2d4c9b3c8ffed74d572e4/dimagi/utils/logging.py#L39-L39) instead, @millerdev, @snopoke.